doc/Overview/Mvux/WIP_Overview.md

```diff
@@ -0,0 +1,179 @@
+    }
+}
+```
+
+### Model
+
+The `WeatherModel` is the **Model** in MVUX (this would be referred to as a ViewModel in MVVM):
+
+```c#
+public partial record WeatherModel(IWeatherService WeatherService)
+{
+    public IFeed<WeatherInfo> CurrentWeather => Feed.Async(WeatherService.GetCurrentWeather);
+}
+```
+
+> [!IMPORTANT]  
+> For the code generation to work, mark the Models and entities with the `partial` modifier, and have the Feed properties' access modifier as `public`.  
+You can learn more about partial classes and methods in [this article](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/partial-classes-and-methods).
+
+The `CurrentWeather` property is an `IFeed<WeatherInfo>`. It uses as an asynchronous gateway to the service, and also holds information about the current status of the data.
+
+The MVUX code-generator (the X in MVUX) generates a data-binding-ready clone of the Model (prefixed `Bindable`), it re-generates its properties, and also introduces commands for all of its public methods that meet a certain criteria, which will be discussed later.
+
+### View
+
+A special control, the `FeedView` is used to display data exposed as an `IFeed<T>`, and provides different display templates for the various statuses of the Feed (e.g. loading, refreshing, error etc.).
+
+The `FeedView` is data-bound to the `CurrentWeather` property which aligns with the `CurrentWeather` property exposed on the `WeatherModel`:
+
+```xml
+<Page x:Class="WeatherApp.MainPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:mvux="using:Uno.Extensions.Reactive.UI">
+
+	<mvux:FeedView Source="{Binding CurrentWeather}">
+		<DataTemplate>
+			<StackPanel>
+				<TextBlock DataContext="{Binding Data}"
+						   Text="{Binding Temperature}" />
+				<Button Content="Refresh"
+						Command="{Binding Refresh}" />
+			</StackPanel>
+		</DataTemplate>
+	</mvux:FeedView>
+
+</Page>
+```
+
+In the View's code-behind, we're assigning the page's `DataContext` to the generated clone of the Model:
+
+```csharp
+public sealed partial class MainPage : Page
+{
+    public MainPage()
+    {
+        this.InitializeComponent();
+
+        DataContext = new BindableWeatherModel(new WeatherService());
+    }
+}
+```
+
+Within the template, the `Data` property is used to access the data that was obtained from the service, currently available by the IFeed. In this case it's the most recent `WeatherInfo` result from the service.
+The `Data` property is bound to the `DataContext` of the `TextBlock`, making it possible to bind the `Text` property of the `TextBlock` to the `Temperature` property.
+
+The `Refresh` command is exposed by the `FeedView` and will cause the Feed to requery its source (the service).
+
+> [!NOTE]
+> For the full example see [How to create a feed](xref:Overview.Mvux.HowToSimpleFeed)
```

Comment:
For an example of "Using MVUX to create an app" we haven't seen how to create an app, and we haven't seen much of the functionality described in the trivial app above.
Many of the benefits alluded to in the opening weren't covered in the example, and it's not clear what the code that is shown does in terms of bringing benefits to the reader.